docs/api/perl/kvstore.md

```diff
@@ -0,0 +1,109 @@
+# KVStore API
+
+Topics:
+* [Basic Push and Pull](#basic-push-and-pull)
+* [List Key-Value Pairs](#list-key-value-pairs)
+
+## Basic Push and Pull
+
+Provides basic operation over multiple devices (GPUs) on a single device.
+
+### Initialization
+
+Let's consider a simple example. It initializes
+a (int, NDArray) pair into the store, and then pulls the value out.
+
+```perl
+    pdl> $kv = mx->kv->create('local')
```

Comment:
@mli yes, I believe 'make html' should work, at least I do not see how the files that I added are any different from say the Scala .md pages.
All links work at least on github.
I'll rework all .md pages and remove the indent in a moment.
Thanks.